azure-functions/acbs-function/mappings/facility/helpers/get-exposure-period.js

```diff
@@ -3,19 +3,31 @@ const moment = require('moment');
 const CONSTANTS = require('../../../constants');
 const { formatYear, formatTimestamp } = require('../../../helpers/date');
 
+/**
+ * Evalutes facility's exposure period in months.
+ * If `Unissued` then guarantee month value is returned.
+ * If `Issued` then cover start and end date difference in months.
+ * @param {Object} facility Facility object
+ * @param {String} dealType Deal type `GEF`, `BSS`, `EWCS`
+ * @returns {Integer} Exposure period in months
+ */
 const getExposurePeriod = (facility, dealType) => {
-  let coverStartDate;
   const { facilitySnapshot } = facility;
 
+  // GEF
   if (dealType === CONSTANTS.PRODUCT.TYPE.GEF) {
-    return facility.tfm.exposurePeriodInMonths || 0;
+    return facilitySnapshot.hasBeenIssued
```

Comment:
can we define the values separately for this? it will be easier to read
i.e. 
```
const exposurePeriodInMonths = facility.tfm?.exposurePeriodInMonths || 0;
const monthsOfCover = facilitySnapshot?.monthsOfCover || 0;
const { hasBeenIssued } = facilitySnapshot;
return hasBeenIssued ? exposurePeriodInMonths : monthsOfCover;
```

Reply:
Have updated.